proto/client/main.go

```diff
@@ -1,6 +1,14 @@
-package main
+/*package main
+
+
+import (
+	pb "bootcampProject/proto"
+	"context"
+	"google.golang.org/grpc"
+	"log"
+	"time"
+)
 
-/*
 const (
 	address = "localhost:50051"
```

Comment:
same comment about getting this value from env vars.

Reply:
client/main file was used only for test grpc connection. I deleted it